Sources/CoreMetrics/Metrics.swift

```diff
@@ -136,6 +136,10 @@ public class Recorder {
 
     /// Record a value.
     ///
+    /// Recording a value is meant to have "set" semantics, rather than "add" semantics.
+    /// Metrics implementations may choose to provide additional "add" or other operations meant to modify an existing value,
```

Comment:
this seems out of place in the user facing API, maybe move to the Hander API which is aimed for the backend dev?